de-lib/src/main/java/org/iplantc/de/client/services/FileEditorServiceFacade.java

```diff
@@ -17,7 +18,7 @@
      *  @param file desired manifest's file ID (path).
      * @param callback executes when RPC call is complete.
      */
-    void getManifest(File file, AsyncCallback<Manifest> callback);
+    void getManifest(File file, DECallback<Manifest> callback);
```

Comment:
Here's another one to test an edge case: the manifest is through data-info, but it uses the tree-urls service in the background. Most likely we should make a fallback in data-info that causes this to not throw the 503 the whole way upward (and tree-urls is rarely down, it's so tiny), but for now it's something to check.